src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -23,6 +23,9 @@ public static class DiagnosticBuilder
 
         public class DiagnosticBuilderInternal
         {
+            private const string TYPE_INACCURACY_CLAUSE = " If this is an inaccuracy in the documentation, please report it to the Bicep Team.";
+            private static readonly Uri TYPE_UNACCURACY_URI = new("https://aka.ms/bicep-type-issues");
+
```

Comment:
[nit] The convention we've been following for `const` & `static readonly` fields in this codebase is to use PascalCase.

Reply:
fixed.